<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PuzzleWindow</class>
 <widget class="QMainWindow" name="PuzzleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>66</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Simon Tatham's Puzzles</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QtCanvas" name="canvas" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status_Bar"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>396</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <widget class="QMenu" name="menuChoose_Game">
     <property name="title">
      <string>Choose Game</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="menuChoose_Game"/>
    <addaction name="actionRestart"/>
    <addaction name="actionSolve"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuType">
    <property name="title">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuType"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actiontbUndo"/>
   <addaction name="actiontbRedo"/>
   <addaction name="actiontb0"/>
   <addaction name="actiontb1"/>
   <addaction name="actiontb2"/>
   <addaction name="actiontb3"/>
   <addaction name="actiontb4"/>
   <addaction name="actiontb5"/>
   <addaction name="actiontb6"/>
   <addaction name="actiontb7"/>
   <addaction name="actiontb8"/>
   <addaction name="actiontb9"/>
  </widget>
  <action name="actiontbUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actiontbRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actiontb0">
   <property name="text">
    <string>0</string>
   </property>
  </action>
  <action name="actiontb1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actiontb2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actiontb3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actiontb4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="actiontb5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="actiontb6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="actiontb7">
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="actiontb8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="actiontb9">
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionSolve">
   <property name="text">
    <string>Solve</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCanvas</class>
   <extends>QWidget</extends>
   <header>qtcanvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
